packages/perseus/src/perseus-types.ts

```diff
@@ -424,7 +425,7 @@ export type LegacyButtonSets = ReadonlyArray<
 export type PerseusExpressionWidgetOptions = {
     // The expression forms the answer may come in
     answerForms: ReadonlyArray<PerseusExpressionAnswerForm>;
-    // Different buttons sets that can show in the expression. Options are "basic", "basic+div", "trig", "prealgebra", "logarithms", "basic relations", "advanced relations"
+    // Different buttons sets that can show in the expression. Options are "basic", "basic+div", "trig", "prealgebra", "scientific", "logarithms", "basic relations", "advanced relations"
```

Comment:
I would rather see us remove the entire list of options in this comment.  We use TypeScript to define the union of available values and re-declaring them here is just a recipe for getting things out of sync.